scalableminds/webknossos
# Open experimental alignment services to public

![image](https://github.com/user-attachments/assets/10bafa84-7654-4618-8e89-a0600b2b710c)
![image](https://github.com/user-attachments/assets/988d2fd2-46b9-487e-8514-31128391fde2)
![image](https://github.com/user-attachments/assets/5cffa38c-3df6-4906-b4dc-916f80cea460)


### Open Questions
- should we mention that this feature might not be free to use in the future?
- should we limit the amount of sections that can be aligned for now?

### URL of deployed dev instance (used for testing):
- publicalignment.webknossos.xyz

### Steps to test:
- checking screenshots and wording is enough. if you want to see the modal on the dev instance, execute `webknossos.DEV.store.dispatch({ type: "SET_AI_JOB_MODAL_STATE", state: "neuron_inferral"  })` (no worker is configured for the dev instance)

### TODOs:
- [X] extend docs
- [X] adapt alignment banner
- [X] fine-tune styling
- [x] wait for RAM-optimized alignment worker
- [x] remove temporary code changes

### Issues:
- contributes #7955

------
(Please delete unneeded items, merge only when none are left open)
